src/_guides/language/effective-dart/style.md

```diff
@@ -264,7 +265,8 @@ UiHandler
 IoStream
 HTTPRequest
 ID
-Db
+DbIOPort
```

Comment:
Would you want this "bad" example to be `DbIoPort` instead?

Reply:
Done.